aws-api-appsync/src/main/java/com/amplifyframework/api/aws/SelectionSet.java

```diff
@@ -186,6 +187,11 @@ public Builder modelSchema(@NonNull ModelSchema modelSchema) {
             return Builder.this;
         }
 
+        public Builder modelSchemaRegistry(@NonNull ModelSchemaRegistry modelSchemaRegistry) {
+            this.modelSchemaRegistry = modelSchemaRegistry;
```

Comment:
Should we add `Objects.requireNotNull(modelSchemaRegistry)` in the `build()` method below?

Reply:
Sounds good!